packages/ckeditor5-dev-tests/tests/utils/manual-tests/compilescripts.js

```diff
@@ -13,41 +13,116 @@ const sinon = require( 'sinon' );
+		};
+
+		mockery.registerMock( '../glob', stubs.glob );
+		mockery.registerMock( './getwebpackconfig', stubs.getWebpackConfig );
+		mockery.registerMock( '../getrelativefilepath', stubs.getRelativeFilePath );
+		mockery.registerMock( 'webpack', stubs.webpack );
+
+		compileManualTestScripts = require( '../../../lib/utils/manual-tests/compilescripts' );
 	} );
 
 	afterEach( () => {
 		sandbox.restore();
 		mockery.disable();
-		mockery.deregisterAll();
 	} );
 
 	it( 'should compile manual test scripts', () => {
-		const webpackSpy = sandbox.spy( ( config, callback ) => callback() );
-		const getWebpackConfigSpy = sandbox.spy( ( entries, buildDir ) => ( {
-			entries,
-			buildDir
-		} ) );
-
-		mockery.registerMock( '../glob', () => [ 'file1', 'file2' ] );
-		mockery.registerMock( './getwebpackconfig', getWebpackConfigSpy );
-		mockery.registerMock( '../getrelativefilepath', ( x ) => x );
-		mockery.registerMock( 'webpack', webpackSpy );
-		sandbox.mock( path, 'join', ( ...chunks ) => chunks.join( '/' ) );
-
-		const compileManualTestScripts = require( '../../../lib/utils/manual-tests/compilescripts' );
-
-		return compileManualTestScripts( 'buildDir', 'manualTestPattern' ).then( () => {
-			expect( getWebpackConfigSpy.getCall( 0 ).args[ 0 ] ).to.deep.equal( { file1: 'file1', file2: 'file2' } );
-			expect( getWebpackConfigSpy.getCall( 0 ).args[ 1 ] ).to.deep.equal( 'buildDir' );
-			expect( webpackSpy.getCall( 0 ).args[ 0 ] ).to.deep.equal( { buildDir: 'buildDir', entries: { file1: 'file1', file2: 'file2' } } );
-		} );
+		stubs.glob.returns( [ 'file1', 'file2' ] );
+
+		return compileManualTestScripts( 'buildDir', [ 'manualTestPattern' ] )
+			.then( () => {
+				expect( stubs.getWebpackConfig.calledOnce ).to.equal( true );
+				expect( stubs.getWebpackConfig.firstCall.args[ 0 ] ).to.deep.equal( {
+					file1: 'file1',
+					file2: 'file2'
+				} );
+				expect( stubs.getWebpackConfig.firstCall.args[ 1 ] ).to.deep.equal( 'buildDir' );
+
+				expect( stubs.webpack.calledOnce ).to.equal( true );
+				expect( stubs.webpack.firstCall.args[ 0 ] ).to.deep.equal( {
+					buildDir: 'buildDir',
+					entries: {
+						file1: 'file1',
+						file2: 'file2'
+					}
+				} );
+
+				expect( stubs.glob.calledOnce ).to.equal( true );
+				expect( stubs.glob.firstCall.args[ 0 ] ).to.equal( 'manualTestPattern' );
+			} );
+	} );
+
+	it( 'resolves a few entry points patterns', () => {
+		const manualTestScriptsPatterns = [
+			'ckeditor5-build-classic/tests/**/manual/**/*.js',
+			'ckeditor5-editor-classic/tests/manual/**/*.js'
+		];
+
+		stubs.glob.onFirstCall().returns( [
+			'ckeditor5-build-classic/tests/manual/ckeditor.js',
+			'ckeditor5-build-classic/tests/manual/ckeditor.compcat.js'
+		] );
+
+		stubs.glob.onSecondCall().returns( [
+			'ckeditor5-editor-classic/tests/manual/classic.js'
+		] );
+
+		return compileManualTestScripts( 'buildDir', manualTestScriptsPatterns )
+			.then( () => {
+				expect( stubs.glob.calledTwice ).to.equal( true );
```

Comment:
Could it be `sinon.assert.calledTwice( spy or stub )`? It'll give much better results if something goes wrong.